tests/lib/cli.js

```diff
@@ -161,6 +164,25 @@ describe("cli", function() {
         });
     });
 
+    describe("when given a config that is a sharable config", function() {
```

Comment:
This test doesn't look like it's testing anything - I think the test config should enable a rule that still fail so you can test the output.